frontend/src/components/my-grant-profile-page-handler/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Heading, Page, Section } from "@python-italia/pycon-styleguide";
+import { FormattedMessage } from "react-intl";
+
+import { useMyProfileWithGrantQuery } from "~/types";
+
+import { MyGrant } from "~/components/my-grant-profile-page-handler/my-grant";
+import { NoGrant } from "~/components/my-grant-profile-page-handler/no-grant";
+import { MetaTags } from "../meta-tags";
+
+export const MyGrantProfilePageHandler = () => {
+  const {
+    data: {
+      me,
```

Comment:
nitpick, but if you only use grant you can just do `me: { grant }`

Reply:
yeah thought would have explode if the grant was None, okay